experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -113,7 +121,13 @@ export interface TreeNode extends Tree<TreeSchema> {
 	 */
 	readonly type: TreeSchemaIdentifier;
 
-	[Symbol.iterator](): Iterator<TreeField>;
+	/**
+	 * Iterate through all fields in the node.
+	 *
+	 * @remarks
+	 * This node may not be mutated during iteration.
+	 */
+	boxedIterator(): IterableIterator<TreeField>;
```

Comment:
This is overriding a member of a base type, so I don't think you need the doc comment here.

That said I like this comment more than the one on Tree, so maybe move this phrasing there (but change node to document, since I it holds onto a cursor and thus will error even if you modify something unrleated)

Reply:
I'd like it to be clear this is an override. TypeScript won't let you put "override" here, so I've been using lack of doc comments to indicate that (since everything is documented, and hovering it shows what its overriding if you don't have a doc comment). The types alone do a pretty good job of being more specific.

Inherit doc would do the same thing: make it clear there is something on the base type this is related to.

Really I'd like TypeScript provide something like "override: for interface use, like "implements" or "specializes", but I don't think such a thing exists sadly.

Reply:
Okay, I removed the doc for now.